catppuccin-steam.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash 
```

Comment:
regression to `#!/bin/bash`
```suggestion
#!/usr/bin/env bash
```